app/charts/combo/axis-height-linear-dual.tsx

```diff
@@ -73,8 +76,8 @@ export const AxisHeightLinearDual = (props: AxisHeightLinearDualProps) => {
         width={axisTitleWidth + TITLE_HPADDING * 2}
         height={
           (isOverlapping
-            ? axisLabelFontSize * Math.ceil(overlapAmount) + TITLE_VPADDING
-            : axisLabelFontSize + TITLE_VPADDING) *
+            ? axisLabelHeight * Math.ceil(overlapAmount) + TITLE_VPADDING
+            : axisLabelHeight + TITLE_VPADDING) *
             2 +
           TOP_MARGIN
```

Comment:
If we add `width` as above (`chartWidth / 2`) to `getTextHeight`, I am pretty sure we don't need this logic :) The goal of `getTextHeight` was to get "correct" height, without a need to multiply or divide things anymore 👍

Reply:
@bprusinowski I will try to refactor it, I have an Idea...

Reply:
![image](https://github.com/user-attachments/assets/b82c7f38-6471-4f2c-b3db-8c6ae9919e1f)

The problem is stuff like this. This counts as 3 lines I don't why, but (#) doesnt count as a line for some reason. if log the value it says. 3 lines.

Reply:
But with this we don't need to know the number of lines, as the text would naturally wrap inside the container so we get correct height back, right?

Reply:
the text is within a `foreignObject` which needs the exact height of all lines combined to ensure correct amount of space for the span element.

Reply:
Yes, but here I was thinking about getTextHeight method - the flow would be:
- get width of the label (half of chart width, potentially with some padding),
- get text height, passing the width to the function, so that the text can naturally wrap inside the measurement container,
- set foreignObject height using the text height.

I don't see a reason to get number of lines of the text, did I misunderstand something? 🥹